vars/main.yml

```diff
@@ -339,6 +341,20 @@ pgbackrest_conf:
     - {option: "process-max", value: "2"}
     - {option: "recovery-option", value: "recovery_target_action=promote"}
 #    - {option: "", value: ""}
+
+pgbackrest_srv_conf:
+  global:  # [global] section
+    - {option: "log-level-file", value: "detail"}
+    - {option: "log-path", value: "/var/log/pgbackrest"}
+    - {option: "repo1-type", value: "{{ pgbackrest_repo_type |lower }}"}
+#   - {option: "repo1-retention-full", value: 5} #  Take care, not to fill up your disks with backups
```

Comment:
I think it's worth uncommenting out this parameter, but specify the default value of 1 or 2

> Take care, not to fill up your disks with backups

this comment seems superfluous to me